jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -0,0 +1,6134 @@
+		putFloat(null, offset, value);
+	}
+
+	/**
+	 * Gets the value of the double in memory referenced by offset.
+	 * This is a non-volatile operation.
+	 * 
+	 * @param offset specifies where to retrieve value in memory
+	 * @return double value stored in memory
+	 */
+	public double getDouble(long offset) {
+		return getDouble(null, offset);
+	}
+
+	/**
+	 * Sets the value of the double at memory offset.
+	 * This is a non-volatile operation.
+	 *  
+	 * @param offset specifies where to retrieve value in memory
+	 * @param value double to store in memory
+	 */
+	public void putDouble(long offset, double value) {
+		putDouble(null, offset, value);
+	}
+
+	/**
+	 * Gets the address value in memory at location of address parameter.
+	 * This is a non-volatile operation.
+	 * 
+	 * @param address specifies where to retrieve the address in memory
+	 * @return address stored in obj
+	 */
+	public long getAddress(long address) {
+		return getAddress(null, address);
+
+	}
+
+	/**
+	 * Sets an address value at the location specified by the address parameter.
+	 * This is a non-volatile operation.
+	 * 
+	 * @param address specifies where to store value in memory
+	 * @param value address to store at address parameter
+	 */
+	public void putAddress(long address, long value) {
+		putAddress(null, address, value);
+	}
+
+	/**
+	 * Allocates a block and adds it to a doubly-linked list for tracking purposes.  Actual size of the block is
+	 * requested size plus space for two pointers.
+	 * If size passed is 0, no memory will be allocated.
+	 * 
+	 * @param size requested size of memory in bytes
+	 * @return starting address of memory
+	 * 
+	 * @throws OutOfMemoryError if no memory can be allocated
+	 * @throws IllegalArgumentException if size is not valid
+	 */
+	public long allocateMemory(long size) {
+		allocateMemoryChecks(size);
+
+		if (0L == size) {
+			return 0L;
+		}
+
+		long address = allocateMemory0(size);
+
+		if (0L == address) {
+			throw new OutOfMemoryError();
+		}
+
+		return address;
+	}
+
+	/**
+	 * Reallocates a block and adds it to a doubly-linked list for tracking purposes.  Actual size of the block is
+	 * requested size plus space for two pointers.
+	 * If size passed is 0, no memory will be allocated.
+	 * 
+	 * @param size requested size of memory in bytes
+	 * @return starting address of memory
+	 * 
+	 * @throws OutOfMemoryError if no memory can be allocated
+	 * @throws IllegalArgumentException if size is not valid
+	 */
+	public long reallocateMemory(long address, long size) {
+		reallocateMemoryChecks(address, size);
+
+		if (0L == size) {
+			freeMemory(address);
+			return 0L;
+		}
+
+		long addressNew;
+		if (0L == address) {
+			addressNew = allocateMemory0(size);
+		} else {
+			addressNew = reallocateMemory0(address, size);
+		}
+
+		if (0L == addressNew) {
+			throw new OutOfMemoryError();
+		}
+
+		return addressNew;
+	}
+
+	/**
+	 * Set object at specified index in obj parameter
```

Comment:
Set the byte at the specified index and size in obj parameter

Reply:
Sentences should end with a period. There are many single line sentences in this class which do not.

Reply:
Are periods required for sentences describing @param?

Reply:
Looking at the OpenJDK javadoc it seems not
https://docs.oracle.com/javase/9/docs/api/java/lang/Appendable.html#append-java.lang.CharSequence-